web/src/filter.js

```diff
@@ -1104,7 +1104,7 @@ export class Filter {
         return [...parts, ...more_parts].join(", ");
     }
 
-    static describe(operators) {
+    static describe_html(operators) {
```

Comment:
Can we add a comment or maybe just a better name -- like should this be `html_description` or `description_as_html`?

Reply:
Yeah I like that!

Reply:
Updated the name